datacapturing/src/androidTest/java/de/cyface/datacapturing/MeasurementContentProviderClientTest.java

```diff
@@ -0,0 +1,129 @@
+package de.cyface.datacapturing;
+
+import android.content.ContentProviderClient;
+import android.content.ContentValues;
+import android.content.Context;
+import android.database.Cursor;
+import android.net.Uri;
+import android.os.Build;
+import android.os.RemoteException;
+import android.support.test.InstrumentationRegistry;
+import android.support.test.filters.MediumTest;
+import android.support.test.runner.AndroidJUnit4;
+import android.util.Log;
+
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import de.cyface.datacapturing.model.GeoLocation;
+import de.cyface.persistence.BuildConfig;
+import de.cyface.persistence.GpsPointsTable;
+import de.cyface.persistence.MeasurementTable;
+import de.cyface.persistence.MeasuringPointsContentProvider;
+import de.cyface.synchronization.MeasurementContentProviderClient;
+
+import static de.cyface.datacapturing.persistence.MeasurementPersistence.MAX_SIMULTANEOUS_OPERATIONS;
+import static de.cyface.persistence.AbstractCyfaceMeasurementTable.DATABASE_QUERY_LIMIT;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+/**
+ * TODO: To avoid circular dependencies I had to place this test into the capturing module instead
```

Comment:
I would probaly have put this into the persistence module. But I guess the capturing module is ok. This is the one responsible for calling the storage API.

Reply:
The Test uses classes from: datacapturing, persistence and synchronization. For this reason I was forces to place it in datacapturing as this is the only module of those which can access all three.